File: 프로그래머스/0/181880. 1로 만들기/1로 만들기.js
function solution(num_list) {
    var answer = 0;
    
    for(let num of num_list) {
        while(num !== 1){
            if(num % 2) num = (num-1) / 2;
            else        num /= 2;
            answer++;
        }
    }
    return answer;
}

File: 프로그래머스/0/181880. 1로 만들기/1로 만들기.py
def is_odd(num):
    return num % 2

def convert_to_1(num):
    cnt = 0
    while num != 1:
        cnt += 1
        if is_odd(num):
            num -= 1
            num //= 2
        else:
            num //= 2
    return cnt

def solution(num_list):
    answer = 0
    
    for num in num_list:
        answer += convert_to_1(num)
    
    return answer